File: naTanjir/naTanjir.Model/RestoranFavorit.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace naTanjir.Model
{
    public class RestoranFavorit
    {
        public int RestoranFavoritId { get; set; }

        public DateTime DatumDodavanja { get; set; }

        public int KorisnikId { get; set; }

        public int RestoranId { get; set; }

        public virtual Korisnici Korisnik { get; set; }

        public virtual Restoran Restoran { get; set; }
    }
}

File: naTanjir/UI/natanjir_mobile/lib/models/restoran_favorit.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:natanjir_mobile/models/korisnici.dart';
import 'package:natanjir_mobile/models/restoran.dart';
part 'restoran_favorit.g.dart';

@JsonSerializable()
class RestoranFavorit {
  int? restoranFavoritId;
  String? datumDodavanja;
  int? korisnikId;
  int? restoranId;
  Korisnici? korisnik;
  Restoran? restoran;
  bool? isFavorit;

  RestoranFavorit();

  factory RestoranFavorit.fromJson(Map<String, dynamic> json) =>
      _$RestoranFavoritFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$RestoranFavoritToJson(this);
}
